vernac/classes.ml

```diff
@@ -112,22 +110,22 @@ let instance_hook k info global imps ?hook cst =
   Typeclasses.declare_instance (Some info) (not global) cst;
   (match hook with Some h -> h cst | None -> ())
 
-let declare_instance_constant k info global imps ?hook id decl poly evm term termtype =
+let declare_instance_constant k info global imps ?hook id decl poly sigma term termtype =
```

Comment:
Here you rename evm -> sigma but the one above you rename evars -> evd

Reply:
More consistency needed indeed. The thing is that I am not so fond of `sigma`, but I recall that this is the proper convention, right @ppedrot ?